### DISCUSSION TITLE
hyphens css property in tailwindcss

### DISCUSSION THREAD

**Discussion by jmhungdev** [tailwindlabs/tailwindcss#2243] [Ideas]
Requesting hyphens: none | manual | auto implementation in tailwindcss 
reference link: https://css-tricks.com/almanac/properties/h/hyphenate/

It goes well with break-words.

**Comment by mashpie** [comment]
Maybe dup of #1846

while it's easy to add on my own... a hyphens utility would complete tailwindcss IMHO - hello DE here, with so talented skills to create very long words quite frequently, ie:

```
Donauschifffahrtspolizeiverordnung
```

now have fun setting that as headline on mobile :)

**Reply by mashpie** [reply]
Ha, found another one :)

```
Rindfleischetikettierungsüberwachungsaufgabenübertragungsgesetz
```

**Comment by fratzinger** [comment]
I've created a PR: https://github.com/tailwindlabs/tailwindcss/pull/10071
